test/e2e_node/user_namespaces_test.go

```diff
@@ -0,0 +1,97 @@
+//go:build linux
```

Comment:
Just curious, why here? We have other e2e tests here: https://github.com/kubernetes/kubernetes/commit/b1eaf6a2d978d460d5d8f274102851925afff53c

I'm completely okay if this test makes sense here, just curious about the reason

Reply:
My goal was to enable the UserNamespacesSupport and UserNamespacesPodSecurityStandards features in the tests, but that does not seem to work with the API server.